packages/@okta/vuepress-site/docs/guides/custom-error-pages/main/index.md

```diff
@@ -47,19 +47,25 @@ The [Brands API](/docs/reference/api/brands/) allows you to set icons, images, a
 
 ## Edit the error page
 
-The HTML editor that is provided on the **Error pages code editor** section of **Customizations** allows you to modify any HTML, CSS, and JavaScript on the error page.
+Use the code editor to modify any HTML, CSS, or JavaScript on the error page. See [Customization examples](#customization-examples) for snippets that you can update and use.
 
-1. In the Admin Console, select **Customizations** and then **Error pages code editor**.
-1. Make changes directly in the embedded **Error Pages** editor.
-1. Click **Preview** to preview your changes before you publish.
-1. Click **Reset to Default** if you need to remove all of your customizations and restore the default HTML/CSS and JavaScript code.
-1. Click **Save and Publish** to commit your changes.
+> **Note:** There is no draft mode in the code editor. When you click **Publish**, your changes go live. If you want to test changes without impacting production, use a test or development org.
 
-> **Note:** See [Customization examples](#customization-examples) for snippets that you can update and use.
+1. In the Admin Console, go to **Customizations** > **Branding**.
+2. In the **Okta-hosted Error Pages box**, click **Edit**.
+3. To open the code editor, turn on the toggle next to **Code editor**.
+4. Make changes directly in the editor. If you type `{`, you see a list of available variables that you can use. See [Use variables](#use-variables).
```

Comment:
some more details re: autocomplete:

For error pages we explicitly trigger on `.`  `{` or `(` 

For emails it can trigger on # ! or { and monaco has smart triggers around the letters which are the start of eligible variable words (for orgLogo, both o and l would trigger autocomplete)

Reply:
Thanks, Albert. I've updated the content.